tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -421,6 +421,12 @@ dualtor/test_tunnel_memory_leak.py::test_tunnel_memory_leak:
     conditions:
       - "https://github.com/sonic-net/sonic-mgmt/issues/11403 and 'dualtor-64' in topo_name"
 
+dualtor_io/test_link_failure.py:
```

Comment:
There are other dualtor_io testcases, should we include them all here?
<img width="482" alt="image" src="https://github.com/sonic-net/sonic-mgmt/assets/35479537/7cf12549-7101-4184-9279-2bd60df90b61">

Reply:
Yes I think so, I added in latest commit and tested